makedoc.g

```diff
@@ -6,12 +6,12 @@
 ##      gap makedoc.g
 ##
 
-PACKAGE := "Semigroups";
 LoadPackage("GAPDoc");
 LoadPackage("semigroups");
 
 SemigroupsMakeDoc();
 CopyHTMLStyleFiles("doc");
-GAPDocManualLab(PACKAGE);
+# The next step loads the Semigroups package!
+GAPDocManualLab("Semigroups");
```

Comment:
The Semigroups package is already loaded on line 10.

Reply:
Right! But it would be possible to build the doc without loading the package, if this line and line 10 weren't included

Reply:
I should remove this comment

Reply:
Ah I see 🙂